tests/acceptance/EditorTextBlockCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Newsletter;
+
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+
+class EditorTextBlockCest {
+  function addText(\AcceptanceTester $I) {
+    $I->wantTo('add Text block to newsletter');
+    $newsletterTitle = 'Text Block Newsletter';
+    $textInEditor = ('[data-automation-id="text_block_in_editor"]');
+    (new Newsletter())
+      ->withSubject($newsletterTitle)
+      ->loadBodyFrom('newsletterWithText.json')
+      ->create();
+    $I->login();
+    $I->amOnMailpoetPage('Emails');
+    $I->waitForText($newsletterTitle);
+    $I->clickItemRowActionByItemName($newsletterTitle, 'Edit');
+    // Create Text block
+    $I->waitForText('Text');
+    $I->wait(1); // just to be sure
+    $I->dragAndDrop('#automation_editor_block_text', '#mce_1');
```

Comment:
I created a newsletter with the same test content manually but couldn't find the element with id `mce_1`. Can you tell me which element is that?

Reply:
It's the first element under the header block, which is #mce_0, or the second content area in the template.